ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
 </server>
 ----
 
-This example `server.xml` configures the server to support Java Server Pages 2.3,
-to listen to incoming traffic to localhost on port 9080, and to automatically
-expand WAR files when they are deployed.
+This `server.xml` file configures the server to do the following things:
 
-The `server.xml` is described in more detail <<server-xml,below>>. When
-discussing config for {projectName}, the term "server config" can be used to refer
-to all of the files that make up the server config, or specifically to the
-config held in XML files; it is usually clear in context and, if not, the
-phrase "server XML config" might be used to clarify. The next few sections
-describe each of the config files in the order they are processed.
+* Support the Java Server Pages 2.3 feature
+* Listen to incoming traffic to `localhost` on port `9080`
+* Automatically expand WAR files when they are deployed
 
-== server.env
-The `server.env` files are optional but, when present, are read by the
-`bin/server` script and specify environment variables primarily used to
-influence the behavior of the `bin/server` script. `server.env` files are read
-from the following locations in order:
+The term _server config_ can be used to refer to all of the files that make up the server configuration, or specifically to the configuration that's in the XML files. If it's not clear in context, you can use the term _server XML config_ to refer to the configuration in the XML files.
 
-* `${wlp.install.dir}/etc/`
-* `${wlp.user.dir}/shared/`
-* `${server.config.dir}/`
+== Configuration files
 
-If the same property is set in multiple locations the last value found is
-used.
+The server config files are processed in the following order:
 
-The most common use of this file is to set the following settings:
+. <<server-env,`server.env`>>
+. <<jvm-options,`jvm.options`>>
+. <<bootstrap-properties,`bootstrap.properties`>>
+. <<server-xml,`server.xml`>>
 
-* `JAVA_HOME` - indicates which JVM to use. If this is not set, the system default
-  is used which often isn't desired.
-* `WLP_USER_DIR` - indicates the location of the `usr` directory which contains
-  the server config. This can only be set in the `etc/server.env` file because
-  the other locations are relative to the `usr` directory.
-* `WLP_OUTPUT_DIR` - indicates where the server should write files to. By
-  default, the server writes to the directory structure that the config is
-  read from. However, in some secure profiles the server config needs to
-  be read-only and so the server must write files to another location.
+[#server-env]
+=== `server.env`
```

Comment:
I don't like using `` around the title. It makes the styling of the headings look terrible.

Reply:
Updated